ansible.cfg

```diff
@@ -0,0 +1,3 @@
+[defaults]
+
+roles_path = provisioning/roles
```

Comment:
Ansible config applied in this order

* ANSIBLE_CONFIG (an environment variable)
* ansible.cfg (in the current directory)
* .ansible.cfg (in the home directory)
* /etc/ansible/ansible.cfg

Source: http://docs.ansible.com/ansible/intro_configuration.html